crates/rust-project-goals/src/gh/issues.rs

```diff
@@ -193,14 +193,24 @@ pub fn create_issue(
         .output()?;
 
     if !output.status.success() {
-        Err(Error::str(format!(
+        return Err(Error::str(format!(
             "failed to create issue `{}`: {}",
             title,
             String::from_utf8_lossy(&output.stderr)
-        )))
-    } else {
-        Ok(())
+        )));
     }
+
+    // Output in stdout looks like
+    //
+    // https://github.com/rust-lang/rust-project-goals/issues/413}
```

Comment:
I'm not sure I understand, is this saying that it will print "https://github.com/rust-lang/rust-project-goals/issues/413}"?